.github/workflows/release.yml

```diff
@@ -92,4 +149,8 @@ jobs:
         if [[ "$ORT_SERVER_VERSION" =~ -RC[0-9]+ ]]; then
           PRERELEASE_ARG="--prerelease"
         fi
-        gh release create $ORT_SERVER_VERSION --notes-file release-notes/RELEASE_NOTES.md $PRERELEASE_ARG
+
+        gh release create $ORT_SERVER_VERSION --notes-file ./artifacts/release-notes/RELEASE_NOTES.md $PRERELEASE_ARG \
+          './artifacts/osc-cli-linux-x64/osc-cli-linux-x64.tar.gz#osc-cli-linux-x64' \
+          './artifacts/osc-cli-macos-arm64/osc-cli-macos-arm64.tar.gz#osc-cli-macos-arm64' \
+          './artifacts/osc-cli-macos-x64/osc-cli-macos-x64.tar.gz#osc-cli-macos-x64'
```

Comment:
So in the end, there is no Windows release ?

Reply:
See https://github.com/eclipse-apoapsis/ort-server/pull/2101. 
My plan is, to also offer Windows binaries.